communications/models.py

```diff
@@ -1,3 +1,26 @@
 from django.db import models
 
-# Create your models here.
+
+class ChatRoom(models.Model):  # Create your models here.
+    id = models.AutoField(primary_key=True)
+    company_id_1 = models.ForeignKey(
+        "companies.CompanyProfile",
+        on_delete=models.SET_NULL,
+        related_name="company1",
+        null=True,
+    )
+    company_id_2 = models.ForeignKey(
+        "companies.CompanyProfile",
+        on_delete=models.SET_NULL,
+        related_name="company2",
+        null=True,
+    )
+    mongo_room_id = models.CharField(max_length=255)
```

Comment:
If you're syncing with a MongoDB collection, consider making the field unique and adding a comment:
```
mongo_room_id = models.CharField(max_length=255, unique=True, help_text="MongoDB Room ObjectId or key")
```